discovery-provider/src/tasks/index.py

```diff
@@ -122,6 +125,9 @@ def index_blocks(self, db, blocks_list):
             former_current_block.is_current = False
             session.add(block_model)
 
+            # add the current block to redis
+            redis.set(most_recent_indexed_block_redis_key, block.number)
```

Comment:
i realized it's not clear that this keyname "most_recent_indexed_block_redis_key" is blocknumber vs blockhash, since there's another rediskey "latest_block_hash_redis_key". worth clarifying?  prob not in this pr

Reply:
any suggestion to change it to? i'm in the code right now

Reply:
lets do this later